packages/cli/template/plugin/__tests__/index.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * @jest-environment-options {"requireTemplate": true}
+ */
+
+/// <reference types="@brandingbrand/code-jest-config" />
+
+import plugin from "../src";
+
+describe("plugin", () => {
```

Comment:
using generated plugin in example gives prettier rule warning Strings must use single quote, we should probably switch to single quotes here to match rule

Reply:
after looking into it - we will need to add the prettier config at the root of the repo so vs code can sync those settings appropriately with format command. we may want to do this is subsequent commit after merge since it affects most files.